importer/load.py

```diff
@@ -48,50 +30,52 @@ def main(argv):
                         type=str,
                         metavar="FILE")
     parse_result = parser.parse_args(argv)
-    FILE = parse_result.file
 
-    driver = GraphDatabase.driver("bolt://"+URL, auth=basic_auth(USER, PASS))
+    driver = GraphDatabase.driver("bolt://"+url, auth=basic_auth(usr, pwd))
```

Comment:
As discussed, we'll keep this for now. Though, space between the operator and the operands.